benchmark/CMakeLists.txt

```diff
@@ -75,3 +75,7 @@ ADD_BENCH(timings-eigen)
 IF(URDFDOM_FOUND AND HPP_FCL_FOUND)
   ADD_BENCH(timings-geometry TRUE)
 ENDIF(URDFDOM_FOUND AND HPP_FCL_FOUND)
+
+# timings-jacobian
+#
+ADD_BENCH(timings-jacobian TRUE)
```

Comment:
Jacobiand are already bench in the main timings file.
Maybe it is not needed to over populate this directory

Reply:
That benchmark only has `computeJointJacobians(model,data,q)` along with many unrelated algorithms. The purpose of this benchmark is specifically comparing the speed of the different ways of computing the Jacobian (mostly, `computeJointJacobians` + `getJointJacobian` vs `jointJacobian`).
It may have meaningful results. For instance, on my machine, it seems `forwardKinematics` + `jointJacobian` is already a bit slower than `computeJointJacobians` (which also computes `forwardKinematics` as a side effect) + `getJointJacobian`. In other words, if, other than the Jacobian, you also need the result of `forwardKinematics`, then it is always better to use `computeJointJacobians`.
I may remove it if you want, but I think it is useful

Reply:
Yes, you can not compare `forwardKinematics` + `jointJacobian` vs `computeJointJacobians` because this is not the same purpose. In general, we do not need at all `jointJacobian`.
But these results are already known.